custom/conf/app.example.ini

```diff
@@ -220,6 +220,9 @@ TIMEOUT_STEP = 10s
 ; This setting determines how often the db is queried to get the latest notification counts.
 ; If the browser client supports EventSource, it will be used in preference to polling notification.
 EVENT_SOURCE_UPDATE_TIME = 10s
+USE_SHARED_WORKER=true ; use a eventsource in shared worker
+USE_WORKER=false ; use eventsource in worker
+USE_PLAIN_EVENT_SOURCE=false ; allow direct use of event source outside of worker
```

Comment:
I'd say make one pref `USE_POLLING`, defaulting to `false`.

Reply:
I've dropped the other options. You can disable the eventsource by setting the update time to negative.

Reply:
Should disable on zero or less `EVENT_SOURCE_UPDATE_TIME` and update docs to this new behaviour.

Reply:
already done.